examples/gno.land/r/x/manfred_upgrade_patterns/upgrade_g/logic/v1/counter.gno

```diff
@@ -0,0 +1,29 @@
+package v1
+
+import (
+	"std"
+
+	"gno.land/r/x/manfred_upgrade_patterns/upgrade_g/admin"
+)
+
+var store admin.Store = nil
+
+func Init() {
```

Comment:
Is this uppercase on purpose?

Reply:
It needs to be lowercase if this is supposed to be a realm constructor

Reply:
Not a constructor, meant to be called by the testing code as an emulated deployment